### ISSUE TITLE
Create 6.2 Build

### ISSUE THREAD

**Issue by Sdaswani** [mozilla-mobile/focus-android#2969]
- [ ] Create 6.2 branch / tag
- [ ] Cherry Pick Find-in-page integration https://github.com/mozilla-mobile/focus-android/issues/2690
- [ ] Cherry Pick GV Switcher https://github.com/mozilla-mobile/focus-android/issues/2915
- [ ] Cherrry Pick A/B Test https://github.com/mozilla-mobile/focus-android/issues/2916

@boek assigning you to play traffic cop.

**Comment by Sdaswani** [comment]
cc @colintheshots @ekager

**Comment by Sdaswani** [comment]
> - From what branch are we releasing?

@boek ?  I think from the 6.1 tag?  

> - Why now from master? Landing the GV switch but not other related GV changes calls for trouble (especially while we are still facing issues in Nightly).

Because 7.0 / master is not in a shippable state.

**Comment by Sdaswani** [comment]
@pocmo can you expound more on this:

> Landing the GV switch but not other related GV changes calls for trouble (especially while we are still facing issues in Nightly).

**Comment by pocmo** [comment]
* Didn't @ekager land a bunch of other GV fixes in master that we'd would want in Beta too?
* We updated GV frequently on master - do we need to cherry-pick something there too?
* After landing #2915 we got reports that some Nightly users are back on WebView. Are we sure everything's working in Nightly correctly before we go to Beta?

There are 115 commits in master that are not in 6.1. I hope we do not miss important pieces and burn our Beta audience:
https://github.com/mozilla-mobile/focus-android/compare/releases/v6.1...master
